specification/network/resource-manager/readme.python.md

```diff
@@ -1,363 +1,20 @@
-
-``` yaml $(tag) == 'package-2019-04'
-namespace: azure.mgmt.network.v2019_04_01
-output-folder: $(python-sdks-folder)/network/azure-mgmt-network/azure/mgmt/network/v2019_04_01
-```
-
-### Tag: package-2019-02 and python
-
-These settings apply only when `--tag=package-2019-02 --python` is specified on the command line.
-Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
-
-``` yaml $(tag) == 'package-2019-02'
-namespace: azure.mgmt.network.v2019_02_01
-output-folder: $(python-sdks-folder)/network/azure-mgmt-network/azure/mgmt/network/v2019_02_01
-```
-
-### Tag: package-2018-12 and python
-
-These settings apply only when `--tag=package-2018-12 --python` is specified on the command line.
-Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
-
-``` yaml $(tag) == 'package-2018-12'
-namespace: azure.mgmt.network.v2018_12_01
-output-folder: $(python-sdks-folder)/network/azure-mgmt-network/azure/mgmt/network/v2018_12_01
-```
-
-### Tag: package-2018-11 and python
-
-These settings apply only when `--tag=package-2018-11 --python` is specified on the command line.
-Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
-
-``` yaml $(tag) == 'package-2018-11'
-namespace: azure.mgmt.network.v2018_11_01
-output-folder: $(python-sdks-folder)/network/azure-mgmt-network/azure/mgmt/network/v2018_11_01
-```
-
-### Tag: package-2018-04 and python
-
-These settings apply only when `--tag=package-2018-04 --python` is specified on the command line.
-Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
-
-``` yaml $(tag) == 'package-2018-04'
-namespace: azure.mgmt.network.v2018_04_01
-output-folder: $(python-sdks-folder)/network/azure-mgmt-network/azure/mgmt/network/v2018_04_01
-```
-
-### Tag: package-2017-10 and python
-
-These settings apply only when `--tag=package-2017-10 --python` is specified on the command line.
-Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
-
-``` yaml $(tag) == 'package-2017-10'
-namespace: azure.mgmt.network.v2017_10_01
-output-folder: $(python-sdks-folder)/network/azure-mgmt-network/azure/mgmt/network/v2017_10_01
-```
-
-### Tag: package-2015-06split and python
-
-These settings apply only when `--tag=package-2015-06split --python` is specified on the command line.
-Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
-
-``` yaml $(tag) == 'package-2015-06split'
-namespace: azure.mgmt.network.v2015_06_15
-output-folder: $(python-sdks-folder)/network/azure-mgmt-network/azure/mgmt/network/v2015_06_15
+``` yaml $(python)
+no-namespace-folders: true
+output-folder: $(python-sdks-folder)/network/azure-mgmt-network/azure/mgmt/network
```

Comment:
@ChenxiJiang333 Let's hold on network until other packages are done.

Reply:
got it